python/paddle/distributed/auto_parallel/interface.py

```diff
@@ -228,6 +231,13 @@ def __call__(self, *args, **kwargs):
                 else:
                     output = self._op(*args, **kwargs)
 
+            if paddle.framework.in_pir_mode():
+                block = paddle.static.default_main_program().global_block()
+                rc_end_id = len(block.ops)
+                for idx in range(rc_begin_id, rc_end_id):
+                    rc_op = block.ops[idx]
+                    rc_op.set_int_attr("fwd_recompute_id", _g_recompute_idx)
```

Comment:
Why use the prefix "fwd" ?

Reply:
`fwd_compute_id` indicates the checkpoint information in the forward that needs to be recomputed.
`bwd_recompute_id` corresponds to `fwd_recompute_id` and is newly added in the backward for recompute, which facilitates debugging.
There are comments in the code to provide explanations.